tcpclient/connection.go

```diff
@@ -71,7 +71,7 @@ func (c Connection) String() string {
 
 func (c Connection) GetTCPProcessingDuration() time.Duration {
 	switch c.status {
-	case ConnectionEstablished:
+	case ConnectionEstablished, ConnectionClosed:
```

Comment:
this logic of "connections in status X and Y are actually in the group A" could be exposed in a public method like "connection.wentOk()(bool)", and this connects to other mtcpclient needs (and makes connection stages less coupled to the caller)